src/app/ReadClient.cpp

```diff
@@ -189,25 +188,25 @@ CHIP_ERROR ReadClient::SendReadRequest(ReadPrepareParams & aReadPrepareParams)
     return err;
 }
 
-CHIP_ERROR ReadClient::SendStatusReport(CHIP_ERROR aError)
+CHIP_ERROR ReadClient::SendStatusResponse(CHIP_ERROR aError)
```

Comment:
This API defeats the point of having rich IM status codes. This should have an argument passed in that is of type `InteractionModel::ProtocolCode`. Follow-up issue please.

Reply:
agree, thanks, https://github.com/project-chip/connectedhomeip/issues/9731